src/models/user.ts

```diff
@@ -0,0 +1,17 @@
+import * as Joi from "joi";
+
+export type User = {
```

Comment:
In my opinion interface is better here and I also use I prefix in the interface case

Reply:
Changed. Thank you.